src/Viewer.cpp

```diff
@@ -3,19 +3,34 @@
 void StdViewer::drawBoard(const Gameboard& gameboard) {
   for (int i = 0; i < gameboard.getWidth(); i++) {
     for (int j = 0; j < gameboard.getHeight(); j++) {
-      if (gameboard.getFieldAt(i, j).flagability == Flagability::marked) {
-        std::cout << " F";
-      } else if (gameboard.getFieldAt(i, j).visibility == Visibility::covered) {
-        std::cout << " X";
+      if (gameboard.getFieldAt(i, j).visibility == Visibility::covered) {
+        if (gameboard.getFieldAt(i, j).flagability == Flagability::marked) {
+          std::cout << " F";
```

Comment:
Every literal magic symbol should be extracted to some const variable with meaningful name

Reply:
done